### ISSUE TITLE
scary fftw warnings during compile

### ISSUE THREAD

**Issue by denizyuret** [JuliaImages/Images.jl#687]
I think these are harmless, but I am wondering (1) am I the only one getting these, and (2) if not, how can we fix them.

```
INFO: Recompiling stale cache file /dev/shm/dyuret/lib/v0.6/Images.ji for module Images.
WARNING: --output requested, but no modules defined during run
WARNING: The call to compilecache failed to create a usable precompiled cache file for module FFTW. Got:
WARNING: Cache file "/dev/shm/dyuret/lib/v0.6/FFTW.ji" not found.
WARNING: eval from module Main to ImageCore:    
Expr(:call, Expr(:., :Base, :include_from_node1)::Any, "/dev/shm/dyuret/v0.6/FFTW/src/FFTW.jl")::Any
  ** incremental compilation may be broken for this module **

WARNING: --output requested, but no modules defined during run
WARNING: The call to compilecache failed to create a usable precompiled cache file for module FFTW. Got:
WARNING: Cache file "/dev/shm/dyuret/lib/v0.6/FFTW.ji" not found.
WARNING: eval from module Main to FFTViews:    
Expr(:call, Expr(:., :Base, :include_from_node1)::Any, "/dev/shm/dyuret/v0.6/FFTW/src/FFTW.jl")::Any
  ** incremental compilation may be broken for this module **
```

**Comment by Evizero** [comment]
These warnings are a general julia nuisance currently, so its not specific to Images.jl. As far as I know these warnings are harmless

**Comment by Evizero** [comment]
I am afraid I can't really find an issue discussing this anywhere. I think it was @ararslan who said on Slack that it is safe to ignore (?)

**Comment by timholy** [comment]
Yes, safe to ignore. Will be fixed by using 0.7. Background in https://github.com/JuliaMath/FFTW.jl/issues/17
